src/objects/Address__c-Address Layout.layout

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Thanks, though this file needs to be in the src/layouts folder. You'll see there is already a layout file there. That file just needs to be updated to include the new fields.